src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -1210,17 +1210,129 @@ bool ViewProviderSketch::mouseButtonPressed(int Button, bool pressed, const SbVe
     }
     // Right mouse button ****************************************************
     else if (Button == 2) {
-        if (!pressed) {
+        if (pressed) {
+            // Do things depending on the mode of the user interaction
+            switch (Mode) {
+                case STATUS_NONE: {
+                    bool done = false;
```

Comment:
What did you intend for this (currently unused) variable?

Reply:
removed it